module-2/backend/week_two.md

```diff
@@ -5,19 +5,39 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
+  - It is a file that contains instructions on how to build the schema
 2. What is a migration?
+  - It is a process that runs a series of files to create the table structure
 3. How does a table relate to a model?
+  - 1:1 -> each table should have a model
 4. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
+  - class methods that define the relationships between tables
```

Comment:
we talked about this a little in class today - this is probably an inaccurate question. 

belongs to and has many define instance methods for us that describe the relationships between our models that we can use 👍